### PR TITLE
Fixed AssimpJSONLoader example.

### PR DESCRIPTION
**Pull request by tetuyoko**
Fixed Uncaught ReferenceError: arr is not defined

### CODE REVIEW

File: examples/js/loaders/AssimpJSONLoader.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ THREE.AssimpJSONLoader.prototype = {
 
 				function makeColor(start) {
 					var col = new THREE.Color( );
-					col.setRGB( arr[0], arr[1], arr[2] );
+					col.setRGB( start[0], start[1], start[2] );
```
Comment (WestLangley) [inline examples/js/loaders/AssimpJSONLoader.js]: I think a better solution would be to leave it as-is and just change the function signature:

    function makeColor( arr ) {

A better solution would be to remove the function and make use of the method `Color.fromArray( array )`.

Comment (tetuyoko) [inline examples/js/loaders/AssimpJSONLoader.js]: I think better too. thanks! fixed it.

ff07dbe

**Comment by WestLangley** [comment]
This PR does not look correct to me. Can you provide a link to your test case?

File: examples/js/loaders/AssimpJSONLoader.js [hunk 2]
```diff
@@ -163,9 +157,9 @@ THREE.AssimpJSONLoader.prototype = {
 					b = face.b * 4;
 					c = face.c * 4;
 					face.vertexColors = [
-						makeColor( a ),
-						makeColor( b ),
-						makeColor( c )
+						color.fromArray( a ),
+						color.fromArray( b ),
+						color.fromArray( c )
```
Comment (mrdoob) [inline examples/js/loaders/AssimpJSONLoader.js]: This should be:

```javascript
new THREE.Color().fromArray( a ),
new THREE.Color().fromArray( b ),
new THREE.Color().fromArray( c )
```

Comment (WestLangley) [inline examples/js/loaders/AssimpJSONLoader.js]: @mrdoob  This looks better to me, but it is a guess:

    new THREE.Color().fromArray( in_color[ a ] )

Is `json.colors` in RGBA format?

**Comment by mrdoob** [comment]
Fixed the code in the merge.

@tetuyoko could you double check that it works?

**Comment by tetuyoko** [comment]
it works. thanks! @mrdoob

**Pull request merged**